qutebrowser/config/configfiles.py

```diff
@@ -566,12 +566,16 @@ def _handle_error(self, action: str, name: str) -> typing.Iterator[None]:
 
     def finalize(self) -> None:
         """Do work which needs to be done after reading config.py."""
+        with self._handle_error('loading', 'autoconfig'):
+            if config.instance.warn_autoconfig:
+                raise configexc.Error(
```

Comment:
I think it'd be better to avoid raising an exception here, for two reasons:

- It's just a bit much redirection to raise `configexc.Error` just to have it handled by the enclosing `self._handle_error`
- We should still run the rest of this function (`self._config.update_mutables()`) if this error occurs

I think something like this should work:

```
if config.instance.warn_autoconfig:
    desc = configexc.ConfigErrorDesc("autoconfig loading not specified", "Your config.py should call either `config.load_autoconfig()` (to load settings configured via the GUI) or `config.load_autoconfig(False)` (to not do so)")
```

(I've also adjusted the text a bit, as I'd expect people who run into this error message to not really know about the details of how to configure qutebrowser - if they did, they wouldn't run into it)

Not sure how it gets displayed in the UI exactly though, so take it with a grain of salt.

Reply:
FWIW, it now get's displayed like this:
![Screenshot_2020-10-10_16:18:53](https://user-images.githubusercontent.com/32643670/95657602-9d86ad80-0b15-11eb-8cb9-e19e5feb64a6.png)